app/views/schools/participants/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<% content_for :title, "Early career teachers and mentors" %>
+<% content_for :title, @participant.full_name %>
```

Comment:
Do we need to worry about sending PPI to GA?

Reply:
good catch!